setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup
+
+setup(
+    name="simpleocr",
```

Comment:
simpleocr is memorable and available on pypi. I think I'll rename the repository after we commit this, to be consistent